### ISSUE TITLE
Fix functional tests failure

### ISSUE THREAD

**Issue by luckyj5** [splunk/docker-logging-plugin#45]
Functional tests are failing randomly in CI setup when running all the tests together. 
But tests are not failing when running them in groups (Splitting the tests in 2 or 3 sets and run them in parallel). Need to investigate and troubleshoot to figure out the root cause and fix the failure.

**Comment by dtregonning** [comment]
@luckyj5 - is this still issue. Update and close accordingly.

**Comment by luckyj5** [comment]
Yes, functional tests were failing randomly and hence were not included as part of the CI pipelines. This needs to be looked at and could be a part of CI migration to CircleCI process. 
@gp510 Can you take a look at this when you work on CircleCI stuff?

**Comment by gp510** [comment]
@dtregonning @luckyj5 - Yes I am looking into this
